eng/pipelines/runtime-staging.yml

```diff
@@ -518,11 +518,41 @@ jobs:
       # Turn off the testing for now, until https://github.com/dotnet/runtime/issues/60128 gets resolved
       # ${{ if eq(variables['isRollingBuild'], true) }}:
       #   # extra steps, run tests
-      #   extraStepsTemplate: /eng/pipelines/common/templates/runtimes/android-runtime-and-send-to-helix.yml
+      #   extraStepsTemplate: /eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml
       #   extraStepsParameters:
       #     creator: dotnet-bot
       #     testRunNamePrefixSuffix: Mono_$(_BuildConfig)
 
+#
+# Special Configurations - Rolling build only
+#
+
+#
+# Build the whole product using Mono for Android and run runtime tests with Android devices
+# Co-op GC 
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    runtimeFlavor: mono
+    platforms:
+    #- ${{ if eq(variables['isRollingBuild'], true) }}:
+      - Linux_x64
+    jobParameters:
+      testScope: innerloop
+      nameSuffix: AllSubsets_Mono_CoOp_GC
+      buildArgs: -s mono+libs+libs.tests -c $(_BuildConfig) /p:ArchiveTests=true /p:MonoEnvOptions="MONO_GC_SUSPEND=coop MONO_SLEEP_ABORT_LIMIT=250"
```

Comment:
I think `MonoEnvOptions` sets the `MONO_ENV_OPTIONS` environment variable in the run script (eng/testing/tests.props).  I don't think it sets additional environment variables

Reply:
Can/should I set these at build time?  That would be easier.